src/components/item/ItemMain.tsx

```diff
@@ -50,7 +51,7 @@ const ItemMain = ({ id, children, item }: Props): JSX.Element => {
   const { isChatboxMenuOpen, setIsChatboxMenuOpen } = useLayoutContext();
 
   return (
-    <Box id={id} m={2} className={ITEM_MAIN_CLASS}>
+    <Box id={id} m={2} className={ITEM_MAIN_CLASS} sx={{ height: '100%' }}>
```

Comment:
The `Box` component supports most of the styling properties directly:
```suggestion
    <Box id={id} p={2} className={ITEM_MAIN_CLASS} height="100%">
```

I changed the `margin` to `padding` as it removes the tiny scrolling effect due to combination of the height 100% and the margin.